chargeio/services/PaymentMethods/BankOptions.cs

```diff
@@ -1,56 +1,57 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using Newtonsoft.Json;
-
-namespace ChargeIO
-{
-    public class BankOptions : IPaymentMethod
-    {
-        [JsonProperty("type")]
-        public string Type { get { return "bank"; } }
-
-        [JsonProperty("routing_number")]
-        public string RoutingNumber { get; set; }
-
-        [JsonProperty("account_number")]
-        public string AccountNumber { get; set; }
-
-        [JsonProperty("account_type")]
-        public string AccountType { get; set; }
-
-        [JsonProperty("name")]
-        public string Name { get; set; }
-
-        [JsonProperty("address1")]
-        public string Address1 { get; set; }
-
-        [JsonProperty("address2")]
-        public string Address2 { get; set; }
-
-        [JsonProperty("city")]
-        public string City { get; set; }
-
-        [JsonProperty("state")]
-        public string State { get; set; }
-
-        [JsonProperty("postal_code")]
-        public string PostalCode { get; set; }
-
-        [JsonProperty("country")]
-        public string Country { get; set; }
-
-        [JsonProperty("email")]
-        public string Email { get; set; }
-
-        [JsonProperty("phone")]
-        public string Phone { get; set; }
-
-        [JsonProperty("description")]
-        public string Description { get; set; }
-
-        [JsonProperty("reference")]
-        public string Reference { get; set; }
-    }
-}
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace chargeio
+{
+    [Serializable]
+    public class BankOptions

```

Comment:
Specifically referring to the typeof operation in PaymentMethodConverter within Mapper.cs

https://github.com/affinipay/chargeio-net/commit/843c2cdbd4b71910d7d08729e9a1c337cc2c8307#diff-e2e284f46244802af998d249fbd80be8R96